### ISSUE TITLE
Binary wheels don't support numpy 2.0

### ISSUE THREAD

**Issue by ajb5d** [CarlosBergillos/ts2vg#36]
It looks like the binary wheels for ts2vg don't support numpy 2.0. I'm getting this error when I use ts2vg installed via the macOS wheel:
```
A module that was compiled using NumPy 1.x cannot be run in
NumPy 2.0.0 as it may crash. To support both 1.x and 2.x
versions of NumPy, modules must be compiled with NumPy 2.0.
Some module may need to rebuild instead e.g. with 'pybind11>=2.12'.

If you are a user of the module, the easiest solution will be to
downgrade to 'numpy<2' or try to upgrade the affected module.
We expect that some modules will need time to support NumPy 2.

```

Building from source fixes this issue. I think pushing a new release to pypi is all that is needed. We ran into this in our CI pipeline (https://github.com/neuropsychology/NeuroKit/issues/1009)

**Comment by CarlosBergillos** [comment]
Thanks for reporting! I'll take a look.

**Comment by CarlosBergillos** [comment]
I've just published a new version of ts2vg (1.2.4) to address this (and some other very minor setup improvements).
The wheels for ts2vg 1.2.4 should be compatible with both NumPy 1.xx and NumPy 2.0.
Please check if it's solved.

Unfortunately I've had to drop support for Python 3.6-3.8, since numpy 2.0 is not compatible with those.
Users in Python 3.6-3.8 can continue using ts2vg by using ts2vg<1.2.4 and numpy<2.0.

**Comment by ajb5d** [comment]
Thanks! this is fixed.
